### DISCUSSION TITLE
Setup Step by Step

### DISCUSSION THREAD

**Discussion by haard18** [rjmacarthy/twinny#199] [General]
Can anyone please tell me how to setup this Coding buddy, i am very new to llms and locally running them do i need to 
run any model on ports or something?

**Comment by 3x3cut0r** [comment]
just follow the [twinny quick-start guide](https://rjmacarthy.github.io/twinny-docs/general/quick-start/)

**Reply by BillRaymond** [reply]
@3x3cut0r shared a link that no longer works. Here is the new link: [twinny Quick start](https://twinnydotdev.github.io/twinny-docs/general/quick-start/)
